### DISCUSSION TITLE
Laravel deployment breaks as soon as I moved to the production

### DISCUSSION THREAD

**Discussion by debjit** [vercel/vercel#6062] [Help]
## When I push vercel to the production it's gets stuck to the blank page.
But the dev env does work as expected.
Please share what is happening.
I don't have anywhere to find any clue. Nither I know how to run the PHP artisan command in the instance terminal.
I need help.
```code
**vercel --prod**
```
Note:  I am using laravel and MySQL remote database.

**Comment by f3l1x** [comment]
Hi @debjit. Are you using https://github.com/juicyfx/vercel-php?

**Reply by debjit** [reply]
Yes, I was. When I push it to production it showed a blank screen.
